guide/inheritance-wallet/making-changes.md

```diff
@@ -0,0 +1,321 @@
+    - file: config-changes/change-config-recovery-path
+      alt: Screen showing an enabled recovery path.
+      caption: Bob wants to edit the timelock for the recovery path.
+    - file: config-changes/change-config-recovery-path-initial
+      alt: Screen showing a dialog that lets users edit the unlock time for the recovery path. It is set to 6 months.
+      caption: The current configuration is preselected.
+    - file: config-changes/change-config-recovery-path-changed
+      alt: Screen showing a dialog that lets users edit the unlock delay for the recovery path. It is set to 12 months.
+      caption: Bob changes the timelock to 12 months and saves the changes.
+    - file: config-changes/change-config-recovery-path-done
+      alt: Screen showing an enabled recovery path.
+      caption: The timelock is updated on the recovery path overview screen.
+    - file: config-changes/creation-inheritance-done
+      alt: Screen that lets users define, when the inheritance key set should be unlocked.
+      caption: Bob changes the timelock for the inheritance key set from 6 to 12 months.
+    - file: config-changes/change-config-recovery-review
+      alt: Screen showing the new wallet configuration. The changes that have been made are highlighted.
+      caption: The review screen clearly indicates the changes that Bob has made.
+    - file: config-changes/change-config-success
+      alt: Success screen showing the option to download the wallet backup kit.
+      caption: The new wallet has been created and added to the home screen.
+---
+<!--
+
+Editor's notes
+
+This page covers how users can replace individual keys or change the wallet configuration itself.  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Inheritance-wallet?type=design&node-id=6293%3A21917&mode=design&t=I2e3qgqYRGpAGyaQ-1 
+
+-->
+
+# Making changes 
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+Alice and Bob have been using their savings wallet for a couple of years now and everything has worked smoothly so far. On this page we will take a look at what happens if they need to make changes to their setup. If users are storing bitcoin over a long enough period of time, there are a couple of scenarios that they might run into: 
+
+1. They need to replace a key that has been lost.
+2. They want to change the basic wallet configuration.
+
+### How it works
+
+When users need to make changes to their multisig wallet, such as replacing a key or modifying the wallet configuration, the process involves creating a new wallet and transferring funds. Here's how the process works from Bob's and Alice's perspective:
+
+1. Bob makes the desired changes, which creates a new wallet.
+1. He shares the new wallet with Alice for setup in her application.
+1. Bob and Alice activate the new wallet on all three primary signing devices. 
```

Comment:
Do the kids and Edward also need the new wallet config?

Reply:
No, Alice and Bob intentionally don't give them access to the config, because they don't want their kids to monitor transactions. The config will be stored for them as described in the wallet backup page.